lib/Lib.hs

```diff
@@ -278,8 +271,9 @@ handleTextDocumentHoverRequest = requestHandler STextDocumentHover $ \req ->
   getWsRoot `ekbind` \wsroot ->
     kliftIO $
       withHieDb (wsroot </> ".hiedb") `kbind` \hiedb ->
+        -- TODO: can probably remove this `getDynFlags` call somehow
         kcodensity $
-          getDynFlags wsroot `etbind` \dflags -> retrieveHoverData hiedb dflags req
+          getDynFlags wsroot `etbind` \_ -> retrieveHoverData hiedb req
```

Comment:
It would be nice to eliminate this entirely. I need to look at the types more carefully because removing the `kcodensity` gives an odd error.